spec/requests/movies_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe 'Movies', type: :request do
+  describe 'GET /movies' do
+    context 'when there are records' do
+      before { create_list(:movie, 10) }
+
+      it 'shows 10 movies' do
+        get '/movies',
+            params: {}, as: :json
```

Comment:
i suppose you don't need new line
```suggestion
        get '/movies', params: {}, as: :json
```